test/pseudo-tty/test-tty-color-support.out

```diff
@@ -1,8 +1 @@
 (node:*) Warning: The 'NO_COLOR' env is ignored due to the 'FORCE_COLOR' env being set.
-(node:*) Warning: The 'NO_COLOR' env is ignored due to the 'FORCE_COLOR' env being set.
-(node:*) Warning: The 'NO_COLOR' env is ignored due to the 'FORCE_COLOR' env being set.
-(node:*) Warning: The 'NO_COLOR' env is ignored due to the 'FORCE_COLOR' env being set.
-(node:*) Warning: The 'NODE_DISABLE_COLORS' env is ignored due to the 'FORCE_COLOR' env being set.
```

Comment:
Do we want a separate test checking for the `NODE_DISABLE_COLORS` warning since they've all been removed from here?

Reply:
Ideally it should print both warnings one time, I would think.

Reply:
Sure, I'll add another test for it.